.github/workflows/compressed-size.yml

```diff
@@ -8,18 +8,17 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
```

Comment:
Following the theme, please don't alter things unless absolutely necessary.

Reply:
I think CI is needed to ensure that all works fine in the branch, also it's useful to release new version with changesets

Reply:
[We don't use changesets for publishing](https://github.com/preactjs/signals/blob/main/CONTRIBUTING.md), and again, that fix should be a one-off. We really don't want to be supporting `1.2.x` beyond that

Reply:
Removed commit